hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -262,6 +267,10 @@ export class Insights extends React.Component<Record<string, object>, IInsightsS
     });
   }
 
+  componentWillUnmount(): void {
+    cancelInsightSignal && cancelInsightSignal.abort(); 
```

Comment:
Are we not adding any message here in abort function like we did in heatmap.tsx ?

Reply:
Please refer to https://github.com/apache/ozone/pull/5154#discussion_r1301557777 for the same